python/kserve/pyproject.toml

```diff
@@ -35,7 +35,7 @@ exclude = ["kserve/storage/test"]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 uvicorn = { version = "^0.19.0", extras = ["standard"] }
```

Comment:
uvicorn = { version = "^0.19.0", extras = ["standard"] }

Reply:
@timothyjlaurent Can you help update uvicorn version to "^0.21.1" ？So that PR #3480 can be closed.

Reply:
Sure thing, added it and updated all dependent packages